tests/test_config.py

```diff
@@ -0,0 +1,38 @@
+import pytest
```

Comment:
I still don't see a unit test for: 
- get_default_max_tokens()
- get_default_quality_preset()

Reply:
ie. here's one test (I shouldnt be the one writing these tests out, that was your task)

```
tlm = TLM()
assert tlm._quality_preset = get_default_quality_preset()
```

Reply:
Thanks for the suggestion on `get_default_max_tokens()`.
A `._quality_preset` signaled that it's a private (or hidden) attribute. I don't see it being exposed / used like this in the documentation.
I'll use it for testing purpose.

Reply:
On `get_default_max_tokens()`, I discussed with Hui on implementing the [test 3](https://github.com/cleanlab/cleanlab-tlm/pull/58#pullrequestreview-2796882389).

We figured that this test relies on the model provider's limits / side as they control `max_tokens`. I asked after studying this repo to figure if there's any way I can easily intercept and find truncations.

Reply:
Discovered that `default_max_tokens` is not defined in the repo. Added that to constants and also added this test.

Reply:
i'm not following why it can't be extracted from the `_TLM_MAX_TOKEN_RANGE` variable

Reply:
Our assumption that the default max_output_token would be the max(64, 4096) of 'default' but that's not the case (confirmed by Hui). 
The default is 512 (it's mention in docs) but not defined in this repo.